configuration/pih/pih-config-liberia-harper.json

```diff
@@ -6,39 +6,35 @@
   "site": "HARPER",
 
```

Comment:
This is just the diff that results from replacing the "components" here with those defined in "pih-config-liberia-harper-jjdossen.json"